Examples/SearchExamples/MapsViewController.swift

```diff
@@ -1,11 +1,12 @@
+import UIKit
 import MapboxMaps
 import MapboxSearch
 
-class MapsViewController: UIViewController, ExampleController {
+public class MapsViewController: UIViewController, ExampleController {
     let mapView = MapView(frame: .zero)
     lazy var annotationsManager = mapView.annotations.makePointAnnotationManager()
     
-    override func viewDidLoad() {
+    public override func viewDidLoad() {
```

Comment:
We don't need to make these public, right?

Reply:
Yeah we can remove `public` from this class and function 👍